CHANGELOG.md

```diff
@@ -5,12 +5,16 @@ Notable changes to this project are documented in this file. The format is based
 ## [Unreleased]
 
 Breaking changes:
+- Added support for PureScript 0.14 and dropped support for all previous versions (#12)
 
 New features:
```

Comment:
We've added a note in other PRs that state that role annotations were added. However, that's also part of #12. Not sure whether we want to add a note to that here, or just include it in the PS 0.14 line above.

Otherwise, LGTM.